docs/design/direct-assign-volume.md

```diff
@@ -0,0 +1,139 @@
+# Background 
+
+Today, filesystem volumes are passed in to the guest VM utilizing virtio-fs. In a typical flow, a CSI controller
+will make sure backing storage is attached to the node (often a block device), and then a CSI node driver will
+create the filesystem (if needed) and mount it.
+
+![Volume handling without direct-assignment](arch-images/DA-not-enabled.png)
+
+Ideally, for a VM, we will consume the block device itself and not utilize the host mount. This simplifies volume management
+on the host, and is more efficient from a storage IO perspective. 
+
+![Volume handling with direct-assignment](arch-images/DA-UC.png)
+
+# Proposal - Introduce pattern for directly assigning block device for mounting in the guest
```

Comment:
Does this need to be a proposal doc? Arguably the fact that the doc is part of a PR _is_ the proposal (along with the associated issue) so this PR can just document the design. If this lands as a proposal, you'll have to rework it so that it is no longer (just) a proposal (or write a new version of the doc).

Reply:
Hi @egernst - I've had a change of heart over this. In fact, I've just raised a design proposal doc too, but put the doc into a `proposals/` sub-directory (https://github.com/kata-containers/kata-containers/pull/2062 ;)